ci/infra/testrunner/testrunner.py

```diff
@@ -133,8 +134,14 @@ def main():
     cmd_status = commands.add_parser("status", help="check K8s cluster status")
     cmd_status.set_defaults(func=cluster_status)
 
-    cmd_cluster_upgrade_plan = commands.add_parser("cluster-upgrade-plan", help="plan cluster upgrade")
-    cmd_cluster_upgrade_plan.set_defaults(func=cluster_upgrade_plan)
+    # common parameters for cluster and node upgrade commands
+    upgrade_args = ArgumentParser(add_help=False)
+    upgrade_args.add_argument("-a", "--action", dest="upgrade_action",
+                              help="action: plan or apply upgrade", choices=["plan", "apply"]) 
```

Comment:
there's no `cluster upgrade apply` only `node upgrade apply`

Reply:
Ok, I see let me clarify there's `cluster upgrade plan`   and  `node upgrade plan/apply`?